src/XliffTasks/Model/ResxDocument.cs

```diff
@@ -52,5 +54,23 @@ protected override IEnumerable<TranslatableNode> GetTranslatableNodes()
                     element: valueElement);
             }
         }
+
+        protected override void LoadCore(TextReader reader)
+        {
+            base.LoadCore(reader);
+
+            ExcludeNodesPointingExternalFiles();
+        }
+
+        private void ExcludeNodesPointingExternalFiles()
```

Comment:
Name should be RemoveNodesPointingToExternalFiles. since it actually removes the nodes post-load.

Reply:
Why did you delete my comment?

The method name is just a description of *what* the loop does, not *why* it does it.

There were two things I wanted to convey:

1. Why the nodes cause a problem: they have relative paths to and we save the translated document in another directory.

2. Why we can safely remove them: we don't translate them, and they will still be found at runtime from the main assembly if not included in the satellite assembly.

Reply:
> Why did you delete my comment?

I moved it to test, which is where I prefer it to be. In am ideal world, LUT will be have a dot on the left with the method name from to the code coverage. Or if some one remove the line, the test will fail and point you to why.

A better way would be a simulator that throws when relative path is moved, so there will be more freedom to try different approach without breaking the test.